clang/test/SemaSYCL/check-notdirect-attribute-propagation.cpp

```diff
@@ -0,0 +1,23 @@
+// RUN: %clang %s -fsyntax-only -Xclang -ast-dump -fsycl-device-only | FileCheck %s
```

Comment:
Please don't use the the driver for FE tests. Use` %clang_cc1` instead of `%clang`